airbyte-integrations/connectors/source-github/source_github/spec.json

```diff
@@ -0,0 +1,22 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/github",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Github Source Spec",
+    "type": "object",
+    "required": ["access_token", "repository"],
+    "additionalProperties": false,
+    "properties": {
+      "access_token": {
+        "type": "string",
+        "description": "Log into Github and then generate a <a href=\"https://github.com/settings/tokens\"> personal access token</a>.",
+        "airbyte_secret": true
+      },
+      "repository": {
+        "type": "string",
+        "examples": ["airbytehq/airbyte"],
+        "description": "Space-delimited list of repositories. e.g. airbytehq/airbyte singer-io/tap-github"
```

Comment:
I think this is not relevant at the moment?

Reply:
Done.